UnityProjects/MRTKDevTemplate/Assets/Scripts/SystemKeyboardExample.cs

```diff
@@ -1,163 +1,163 @@
-﻿// Copyright (c) Microsoft Corporation.
-// Licensed under the MIT License.
+﻿// // Copyright (c) Microsoft Corporation.
```

Comment:
Also, was this intentional?

Reply:
No, was just testing some packaging things, didn't mean to commit

Reply:
Fixed